ch_pipeline/core/io.py

```diff
@@ -473,3 +472,52 @@ def next(self, files):
         )
 
         return sorted(list(new_files))
+
+class SaveGaltAutoCorrelation(task.SingleTask):
+    """Extract the autocorrelations of the Galt telescope from a holography acquisition."""
+
+    # YY, YX, XX
+    _galt_prods = [2450, 2746, 3568]
```

Comment:
it would probably be preferable to determine these from the `index_map` rather than hard code them. I don't think these indices would have changed, but in principle they are allowed to.